source/common/filter/http/filter_config_discovery_impl.h

```diff
@@ -52,6 +52,7 @@ class DynamicFilterConfigProviderImpl : public FilterConfigProvider {
 
   FilterConfigSubscriptionSharedPtr subscription_;
   const std::set<std::string> require_type_urls_;
+  absl::optional<Envoy::Http::FilterFactoryCb> config_{absl::nullopt};
```

Comment:
nit: I don't think this needs to be an optional? It can just be a null lambda? Also can we call it `current_config_` or something like that with a comment about what this member is for?

Reply:
I think it's good for consistency with the rest. Lambda pointer is rather confusing and doesn't own the underlying values which is the intent. I'll change the name and comment.